app/Models/User.php

```diff
@@ -2,31 +2,47 @@
 
 namespace App\Models;
 
-use Illuminate\Notifications\Notifiable;
+// use Illuminate\Contracts\Auth\MustVerifyEmail;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
 use Illuminate\Foundation\Auth\User as Authenticatable;
+use Illuminate\Notifications\Notifiable;
+use Laravel\Sanctum\HasApiTokens;
 
 class User extends Authenticatable
 {
-    use Notifiable;
-
+    use HasApiTokens, HasFactory, Notifiable;
```

Comment:
Not sure if these two should be added

Reply:
this was added upstream in version 8.x : https://github.com/laravel/laravel/pull/5663

Reply:
Ok, but is it needed for us?